dbms/src/Common/TiFlashBuildInfo.cpp

```diff
@@ -147,6 +148,14 @@ String getProfile()
     return TIFLASH_PROFILE;
 }
 
+constexpr String getCompilerVersion()
+{
+    return fmt::format(
+        "{} {}",
+        &TIFLASH_CXX_COMPILER[LogFmtDetails::getFileNameOffset(TIFLASH_CXX_COMPILER)],
```

Comment:
TIFLASH_CXX_COMPILER is some strings like "/DATA/disk1/ra_common/tiflash-env-17/sysroot/bin/clang++". Use `LogFmtDetails::getFileNameOffset` to get the "clang++"